rgdps/api/gd/user_comments.py

```diff
@@ -90,13 +112,28 @@ async def like_target_post(
     elif target_type is LikeType.LEVEL:
         result = await likes.like_level(ctx, user.id, target_id, int(is_positive))
 
+    else:
+        raise NotImplementedError
+
     if isinstance(result, ServiceError):
         logger.info(
-            f"Failed to like {target_type!r} {target_id} with error {result!r}.",
+            f"Failed to like/dislike target.",
+            extra={
+                "user_id": user.id,
+                "target_type": target_type.value,
+                "target_id": target_id,
+                "is_positive": is_positive,
+                "error": result.value,
+            },
         )
         return responses.fail()
 
-    logger.info(f"{user} successfully liked {target_type!r} {target_id}.")
+    logger.info(
+        "Successfully liked target.",
```

Comment:
inconsistient log content, above says `like/dislike` and here only `liked`

Reply:
Corrected